# Pololu based motors

Servo_Motor:
  pololu_id: 12
  motor_id: 2
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -1500
    max_pulse: 2000
    max_angle: 1500


DC_Motor:
  pololu_id: 12
  motor_id: 0
  init: 1500
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -50
    max_pulse: 2000
    max_angle: 50

Side_IR:
  pololu_id: 12
  motor_id: 4
  init: 1900
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45
    
Front_IR:
  pololu_id: 12
  motor_id: 6
  init: 1900
  min: 1000
  max: 2000
  reverse: true
  calibration:
    min_pulse: 1000
    min_angle: -45
    max_pulse: 2000
    max_angle: 45

